### PR TITLE
NAS-116958 / 22.02.3 / NAS-116958: Webui in angelfish doesn't expose NFS server count

### PR DESCRIPTION
**Pull request by AlexKarpov98**
Please check for this branch
`NAS-116958-angelfish`

A `NAS-116958` branch was a wrong request that shouldn't merge to a master.

### CODE REVIEW

File: src/app/pages/services/components/service-nfs/service-nfs.component.ts [hunk 1]
```diff
@@ -45,6 +46,8 @@ export class ServiceNfsComponent implements OnInit {
     rpclockd_port: helptext.nfs_srv_rpclockd_port_tooltip,
     udp: helptext.nfs_srv_udp_tooltip,
     userd_manage_gids: helptext.nfs_srv_16_tooltip,
+    servers: helptext.nfs_srv_servers_tooltip,
+    // servers: helptext.nfs_srv_servers_placeholder,
```
Comment (undsoft) [inline src/app/pages/services/components/service-nfs/service-nfs.component.ts:50]: Good idea to review your code before creating a PR 😛

File: src/app/helptext/services/components/service-nfs.ts [hunk 2]
```diff
@@ -10,6 +10,13 @@ export default {
   nfs_srv_allow_nonroot_tooltip: T('Set only if required by the NFS client. Set to allow\
  serving non-root mount requests.'),
 
+  nfs_srv_servers_placeholder: T('Number of servers'),
```
Comment (undsoft) [inline src/app/helptext/services/components/service-nfs.ts:13]: Unused

**Review by undsoft — APPROVED** [review]
The field is there and saves correctly. 👍🏻

File: src/app/pages/services/components/service-nfs/service-nfs.component.ts (hunk 1 above)
Comment (AlexKarpov98) [reply src/app/pages/services/components/service-nfs/service-nfs.component.ts:50]: 👍

**Pull request merged**